src/plugins/condition/ConditionSetCompositionPolicy.js

```diff
@@ -24,9 +24,7 @@ export default function ConditionSetCompositionPolicy(openmct) {
     return {
         allow: function (parent, child) {
             let parentType = parent.type;
-            let childType = child.type;
-
-            if (parentType === 'conditionSet' && childType !== 'condition') {
+            if (parentType === 'conditionSet' && !openmct.telemetry.isTelemetryObject(child)) {
```

Comment:
parent.Type can be inlined since it's referenced only once which helps reduce lines of code. If you still want to keep the parentType variable, please keep the empty line.